src/enr/crypto.ts

```diff
@@ -0,0 +1,55 @@
+import keccak = require("bcrypto/lib/keccak");
+import secp256k1 = require("bcrypto/lib/secp256k1");
```

Comment:
why not enable esModuleInterop since it's default babel behavior? :)

Reply:
I was thinking of just starting with typescript here, and trying w default settings, making more complicated only as necessary.